sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -89,6 +89,11 @@ public partial class HttpAuthorization
         public static bool operator !=(Azure.HttpRange left, Azure.HttpRange right) { throw null; }
         public override string ToString() { throw null; }
     }
+    public partial interface IJsonSerializable
+    {
+        bool TryDeserialize(System.IO.Stream stream, out long bytesConsumed, Azure.SerializableOptions? options = null);
```

Comment:
What are the considerations of using `Stream` here?

I tried to directly use `response.ContentStream`, which is `Stream` as input to this method, and it throws at `JsonDocument.Parse(Stream utf8Json)`. I also tried to read from `response.ContentStream` using `StreamReader`, got empty string from it.
But if I construct `BinaryData` from it with `BinaryData.FromBytes(((MemoryStream)stream).ToArray())`, then JsonDocument.Parse(ReadOnlyMemory<byte> utf8Json) works as expected since `BinaryData` has implicit converter to `ReadOnlyMemory<byte>`. I haven't figured out why it behaves like this since `ContentStream` is abstract in `Response.cs`.

If we use `ReadOnlyMemory<byte>` as input here like before, then we can directly input `response.Content` which is `BinaryData`.